arbiter/arbiter-ui/src/main/java/org/deeplearning4j/arbiter/ui/module/ArbiterModule.java

```diff
@@ -255,30 +257,32 @@ public void onDetach(StatsStorage statsStorage) {
     }
 
     /**
-     * @return Last update time for the page
+     * Return the last update time for the page
      */
-    private Result getLastUpdateTime(){
+    private void getLastUpdateTime(RoutingContext rc){
         //TODO - this forces updates on every request... which is fine, just inefficient
         long t = System.currentTimeMillis();
         UpdateStatus us = new UpdateStatus(t, t, t);
 
-        return ok(Json.toJson(us));
+        rc.response().putHeader("content-type", "application/json").end(Json.encode(us));
```

Comment:
We might want to use our own object mappers here just to keep it consistent? This will use jackson underneath the covers.

Reply:
Fixed